src/bridge/bridge.c

```diff
@@ -682,12 +683,45 @@ main (int argc,
   signal (SIGABRT, cockpit_test_signal_backtrace);
   signal (SIGSEGV, cockpit_test_signal_backtrace);
 
-  /* In case we are run via sshd and we have journald, make sure all
-   * logging output ends up in the journal on *this* machine, not sent
-   * back to the client.
-   */
-  if (g_getenv ("SSH_CONNECTION") && !g_log_writer_is_journald (2) && ! isatty(2))
+  if (g_strv_contains ((const gchar **) argv, "--privileged"))
     {
+      /* We are being spawned, under sudo or pkexec, by the user's copy
+       * of the bridge.  In that case, the first thing that will happen,
+       * if we receive our stderr via the socket that is our stdin.
+       */
+      const char *msg = "\n{\"command\": \"send-stderr\"}";
```

Comment:
Intuitively, I would have expected the "\n" to be at the end here...

Reply:
This is part of the cockpit protocol.  First comes the channel name, then a newline, then the payload.  In the case of control messages, the channel name is empty (ie: newline at the start).

Reply:
Ah, of course, thanks!